composeApp/src/commonMain/kotlin/com/sixbynine/transit/path/widget/WidgetDataFetcher.kt

```diff
@@ -385,12 +396,16 @@ object WidgetDataFetcher {
         alerts: List<Alert>?,
         data: UpcomingDepartures,
         isPathApiBroken: Boolean,
+        isCommuteWidget: Boolean = false,
     ): DepartureBoardData {
         val adjustedStations = stations.toMutableList()
         val stationDatas = arrayListOf<DepartureBoardData.StationData>()
         val avoidMissingTrains = currentAvoidMissingTrains()
 
-        adjustedStations.sortWith(StationComparator(sort, closestStations))
+        if (!isCommuteWidget) {
```

Comment:
I think ignoring sort when it's a commute widget is non-obvious and could be a source of a bug down the line if e.g. you wanted to add some sort of multi-station commute widget where sort was relevant again.

Instead, I think it would be better to either make the sort parameter nullable and pass null from the place that calls for commute widgets